.github/workflows/flutter-build-win.yml

```diff
@@ -0,0 +1,43 @@
+name: Windows WSL Setup CI
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - packages/subiquity_client/**
+      - packages/ubuntu_wizard/**
+      - packages/ubuntu_wsl_setup/**
+      - .github/workflows/**
+  pull_request:
+    branches:
+      - main
+    paths:
+      - packages/subiquity_client/**
+      - packages/ubuntu_wizard/**
+      - packages/ubuntu_wsl_setup/**
+      - .github/workflows/**      
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+    
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - uses: subosito/flutter-action@v1
+      with:
+        channel: 'stable'
+
+    - name: Install Melos
+      run: flutter pub global activate melos
+
+    - name: Bootstrap workspace
+      run: melos bootstrap
```

Comment:
we could skip melos for this job because we only need to build one package. melos would unnecessarily slow it down by running flutter pub get for all packages.

Reply:
Indeed! I thought I'd need melos to bring ubuntu_wizard and subiquity_client, but `flutter pub get` is enough. Thanks for the advice.